client_test.go

```diff
@@ -94,14 +94,22 @@ func TestMain(m *testing.M) {
 	}).Info("running tests against containerd")
 
 	// pull a seed image
-	if _, err = client.Pull(ctx, testImage, WithPullUnpack); err != nil {
-		cmd.Process.Signal(syscall.SIGTERM)
-		cmd.Wait()
-		fmt.Fprintf(os.Stderr, "%s: %s", err, buf.String())
+	if runtime.GOOS != "windows" { // TODO: remove once pull is supported on windows
+		if _, err = client.Pull(ctx, testImage, WithPullUnpack); err != nil {
```

Comment:
We can correct this later but Windows should be able to test the fetch portion by not including `WithPullUnpack`